app/src/components/widgets/multiLevelWidgets/componentHealthCheckTable/columns/nameColumn.jsx

```diff
@@ -30,19 +32,28 @@ export const NameColumn = (
 ) => {
   const color = value.passingRate < minPassingRate ? COLOR_DEEP_RED : COLOR_PASSED;
 
+  const renderComponent = () => {
```

Comment:
Please move it to the separate component in this file and use tooltip f.e. as in `src/layouts/adminLayout/adminSidebar/adminSidebar.jsx` to prevent redundant runtime components creation for each table row.